src/components/Button/Button.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import colors from '../../theme/colors.scss';
+import { SoftKeyConsumer } from '../SoftKey/withSoftKeyManager';
+import './Button.scss';
+
+const prefixCls = 'kai-button';
+
+const Button = React.memo(props => {
+  const {
+    text,
+    icon,
+    iconSide,
+    onClick,
+    focusColor,
+    onFocusChange,
+    index,
+    forwardedRef,
+    softKeyManager,
+  } = props;
+
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocusChange = isNowFocused => {
+    setIsFocused(isNowFocused);
+    if (isNowFocused) {
+      const centerText = 'Select';
+      softKeyManager.setSoftKeyTexts({ centerText });
+      softKeyManager.setSoftKeyCallbacks({
+        centerCallback: onClick,
+      });
+      onFocusChange(index);
+    } else {
+      softKeyManager.unregisterSoftKeys();
+    }
+  };
+
+  const buttonCls = prefixCls;
+  const inputCls = `${prefixCls}-input`;
+  const lineCls = `${prefixCls}-line ${
+    iconSide === 'left' ? 'right' : iconSide === 'right' ? 'left' : ''
+  }`;
+  const textCls = `${prefixCls}-text`;
+  const iconCls = `${prefixCls}-icon-${isFocused ? 'focused' : 'unfocused'}`;
+
+  const iconComp = (
+    <div className={iconCls}>
+      <span className={icon} />
+    </div>
+  );
+
+  return (
+    <div className={buttonCls}>
+      <button
```

Comment:
The default type of this button is submit, but I think  it would be good to change to `reset`and `button`.

Reply:
Good idea